### PR TITLE
Add Desktop version design

### PR DESCRIPTION
**Pull request by mrjuda**
Add Desktop version design to the portfolio website.

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -534,3 +534,203 @@ form .cta-button:hover {
   margin-bottom: 0;
   text-align: center;
 }
+
+/* ##### DESKTOP DESIGN ##### */
+
+/* ##### DESKTOP DESIGN ##### */
+
+/* ##### DESKTOP DESIGN ##### */
+
```
Comment (RolandM99) [inline styles/main.css:543]: - [x] To make your design looks very similar to the Figma template, Please add the margin-left and margin-right of your works section in the desktop version. you can do it easily by adding the padding property of the `class or id portfolio in your section <--PORTFOLIO-->`

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @mrjuda  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 
- Descriptive PR ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [x] Your README file looks good but you can make it more professional. Why don't you add a solid project description, how to run your project, to them so your README is more informative and professional? I think It would be better to add also the mobile and desktop screenshots to your README so that to make very professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/main.css (hunk 1 above)
Comment (mrjuda) [reply styles/main.css:543]: done

**Review by ReshyResh — APPROVED** [review]
# Hi @mrjuda , 

### Good job addressing the changes requested by the previous reviewer!
#### Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**